app/views/branches/health.html.haml

```diff
@@ -26,6 +26,32 @@
       %span.number= rate
       %span.label pass rate on first try
 
+  .stats
+    %h2.subheader= "Build statistics"
+    %p= "Build created on #{@first_built_date.strftime("%Y-%m-%d")}"
+    %table.build-stats
+      %thead
+        %tr
+          %th
+          %th Total
+          %th Failures
+          %th Pass Rate
+        %tr
+          %td= "All Time (#{@days_since_first_build} days)"
+          %td= @total_build_count
+          %td= @total_failure_count
+          %td= "#{(@total_build_count - @total_failure_count) * 100/@total_build_count}%"
```

Comment:
Above you're calculating failure count by subtraction, and then calculating success count here by subtraction. Wouldn't it make sense to get the success count in the controller, calculate the failure count, and percentages, and everything else, and only do interpolation in the view?

Also, the total, 30, 7 thing feels like a lot of copy-paste. Is there a way to refactor this?